CHANGELOG.md

```diff
@@ -106,3 +108,17 @@ New config features
 
 - Output slightly more information about the COSMOSCatalog() (if any) being used
   as the basis of simulations, at the default verbosity level. (#804)
+- Changed the name of galsim.config.CalculateNoiseVar to the slightly more
+  verbose, but also more readable CalculateNoiseVariance. (#820)
+- Setting config['rng'] is no longer required when manually running commands
+  like `galsim.config.BuildGSObject`.  If you do not care about deterministic
+  pseudo-rngs, then it will just use /dev/urandom for you. (#820)
```

Comment:
Sure.  I can add that.  Although, it would require that you are running things with a logger, since that's where config warnings are emitted, and I might guess that most people who neglect to specify the rng might also neglect to pass in a logger.